package.json

```diff
@@ -34,6 +34,8 @@
     "lodash": "^4.17.11",
     "material-ui-pickers": "^1.0.0-rc.17",
     "moment": "^2.22.2",
+    "moment-timezone": "^0.5.23",
```

Comment:
Why do we need this?

Reply:
I just copied the test from the webapp. I now replaced timezone with the regular moment.